pluto_build/python/fastloading.py

```diff
@@ -13,7 +13,6 @@ def ETL(table):
 if __name__ == "__main__":
     tables = [
         "pluto_corrections",
-        "pluto_input_numbldgs",
```

Comment:
So for now, PTS and CAMA get imported from DigitalOcean and the others get loaded from recipes?